modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-single-pvc-template-storage-pool_{context}"]
+= Creating a `pvcTemplate` storage pool in a host path provisioner (HPP) custom resource.
+
+This is useful for when you have one (or a few) large volumes on each node, and you would like to use one to store the hostpath provisioner volumes on. Effectively you are carving up the single large volume into multiple smaller volumes.
+
+If you have a single large persistent volume (PV) on your node, you might want to virtually divide the volume and use one partition to store the volumes only used by HPP. By defining a `storagePool` using a `pvcTemplate` in the HPP custom resource, you can virtually split the PV into multiple smaller volumes, providing more flexibility in data allocation.
+
+A `pvcTemplate` is the `spec` portion of a persistent volume claim (PVC).  For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "iso-pvc"
+  labels:
+    app: containerized-data-importer
+  annotations:
+          cdi.kubevirt.io/storage.import.endpoint: "http://cdi-file-host.cdi:80/tinyCore.iso.tar"
+spec: <1>
+  volumeMode: Block
+    storageClassName: <any_storage_class>
```

Comment:
The indentation is wrong, storageClassName should be on the same level as volumeMode, same goes for accessModes